.travis.yml

```diff
@@ -37,8 +37,9 @@ env:
     - ROS_DISTRO=indigo PRERELEASE=true USE_MOCKUP='industrial_ci/mockups/failing_test' EXPECT_EXIT_CODE=1
     - ROS_DISTRO=kinetic PRERELEASE=true PRERELEASE_REPONAME=industrial_ci
     - ROS_DISTRO=indigo APTKEY_STORE_SKS=hkp://ha.pool.sks-keyservers.vet  # Passing wrong SKS URL as a break test. Should still pass.
-    - ROS_DISTRO=indigo UPSTREAM_WORKSPACE=debian
-    - ROS_DISTRO=indigo UPSTREAM_WORKSPACE=file  # Using default file name for ROSINSTALL_FILENAME
+    - ROS_DISTRO=indigo UPSTREAM_WORKSPACE=debian AFTER_SCRIPT='ccache 2> /dev/null && exit 1; [ "$?" = "127" ]'
+     # Using default file name for ROSINSTALL_FILENAME, test CCACHE, verify cache was filled
+    - ROS_DISTRO=indigo UPSTREAM_WORKSPACE=file CCACHE_DIR=$HOME/.ccache AFTER_SCRIPT='num=($(ccache -s | grep "files in cache")) && (( num[-1] > 0 ))'
```

Comment:
> Locally I get this, which seems not good..:

actually, that's correct behaviour ;)

`$(ccache -s | grep "files in cache")` -> select line from ccache stats
`num=($(ccache -s | grep "files in cache"))` -> define an array with the data (separated by whitespace)
`num[-1]` -> selects the last entry of this array
`(( num[-1] > 0 ))` evaluates an arithmetic expression, the exit code is used as the actual test result

`echo $num` just prints the first item of the array. To print all you have to use `echo "${num[@]}"`.
`echo $?' will print the exit code:
```
$ num=($(ccache -s | grep "files in cache")) && (( num[-1] > 0 ))
$ echo $?
0 # okay
$ num=($(CCACHE_DIR=/tmp ccache -s | grep "files in cache")) && (( num[-1] > 0 ))
$ echo $?
1 # error
```